index.html

```diff
@@ -389,12 +390,80 @@ <h1>Documentation</h1>
   </div>
 
 
-        <!-- mainline releases v4.12.0.6 -->
+        <!-- mainline releases v4.12.0.7 -->
         <div class="wrapper">
         <!--<div class="break"></div>insert this code when a new update comes out to separate a new version from the current.-->
-          <div class="smaller-margin" id="mr-40">
+        <div class="smaller-margin" id="mr-41">
               <h2 class="text-middle">Mainline releases</h2>
               <!-- h2 tag needs to be moved to the newest version -->
+            <h4><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.12.0.6...v4.12.0.7">v4.12.0.7</a></h4>
+          </div>
+
+          <div class="dimmed-text">
+            <p>Release date: 2020-11-27</p>
+          </div>
+
+          <ul>
+              <li>Fixed/added:</li>
+              <ol>
+                  <li>rebased with official coreboot repository commit 957a363</li>
```

Comment:
See my comment: https://github.com/pcengines/coreboot/pull/448#discussion_r528591972
and add nother bulletpoint